src/lib/components/ui/RecentCategoryArticles.svelte

```
@@ -4,6 +4,13 @@
 	import { ArrowRight } from 'lucide-svelte';
 	import Badge from './badge/badge.svelte';
 
+	type Author = {
```

ESLint

[error] 'Author' is defined but never used.

(@typescript-eslint/no-unused-vars)